src/app/src/constants.js

```diff
@@ -0,0 +1,45 @@
+export const POLLING_INTERVAL = 600000;
+
+export const TEMPERATURE = 'TEMPERATURE';
+export const OXYGEN = 'OXYGEN';
+export const PH = 'PH';
+export const TURBIDITY = 'TURBIDITY';
+
+export const VARIABLES = [
+    TEMPERATURE,
+    OXYGEN,
+    PH,
+    TURBIDITY,
+];
+
+export const VARIABLE_DESCRIPTIONS = {
+    [TEMPERATURE]: 'Temperature, water, degrees Celsius',
+    [OXYGEN]: 'Dissolved oxygen, water, unfiltered, milligrams per liter',
+    [PH]: 'pH, water, unfiltered, field, standard units',
+    [TURBIDITY]: 'Turbidity, water, unfiltered, monochrome near infra-red LED light, 780-900 nm, detection angle 90 +-2.5 degrees, formazin nephelometric units (FNU)',
```

Comment:
One additional quick note:

In https://github.com/azavea/stormwater-graphql-research/blob/develop/api/src/schema/types/phlSensors.js I used these values as a unique identifier for each variable. That seemed to work okay, but I do a very thorough job testing that these descriptions were always guaranteed to be the same or that they were getting the correct thing from the sensor response data. I think it's be worthwhile to add some automated tests to verify that the responses are what is expected and that the parsing works as expected.

Additionally: the GraphQL server code above was retrieving all available values for each sensor, which is why I was trying to use these strings for parsing the response.

River Dispatches has an alternate way of getting these by requesting only a smaller subset of the data by:

- sending a specific set of query parameters to the URL, which retrieves only the requested data rather than all of it: https://github.com/azavea/pwd-river-dispatches/blob/develop/usgs-details.html#L194
- parsing the response via array indices rather than using strings like the descriptions above: https://github.com/azavea/pwd-river-dispatches/blob/develop/usgs-details.html#L209

I believe that requesting the subset of data makes the array index parsing possible, since the client would know the order in which is has sent the query parameters and that likely corresponds to the order of the values in the response.

Reply:
Oohhh. Yep I like requesting certain variables way better. Thanks!